src/xai_grok/schemas/complete_request.py

```diff
@@ -0,0 +1,17 @@
+import typing
+
+import pydantic
+
+from xai_grok.schemas.complete_request_metadata import CompleteRequestMetadata
+
+
+class CompleteRequest(pydantic.BaseModel):
+    max_tokens_to_sample: int
+    metadata: typing.Optional[CompleteRequestMetadata] = None
+    model: str
+    prompt: str
+    stop_sequences: typing.List[str] = None
```

Comment:
The `stop_sequences` field is initialized with `None` instead of an empty list `[]`, which can cause type errors when operations are performed on this field expecting a list.